bakerydemo/templates/locations/location_page.html

```diff
@@ -1,111 +1,69 @@
 {% extends "base.html" %}
 {% load wagtailimages_tags navigation_tags %}
 
-{% block head-extra %}
-    <style>
-        /* Following two selectors needed for Google map embed */
-        #map {
-            height: 100%;
-        }
-    </style>
-{% endblock head-extra %}
-
 {% block content %}
-    {% image self.image fill-1920x600 as hero_img %}
-    <div class="container-fluid hero" style="background-image:url('{{ hero_img.url }}')">
-        <div class="hero-gradient-mask"></div>
-            <div class="container">
+{% include "base/include/header-hero.html" %}
+
+<div class="container bread-detail">
+    <div class="row">
+        <div class="col-md-12">
+            <div class="col-md-7">
                 <div class="row">
-                    <div class="col-md-7">
-                        <h1>{{ page.title }}</h1>
-                        <p class="stand-first">
-                            {% if page.is_open %}
-                                This location is currently open
-                            {% else %}
-                                Sorry, this location is currently closed
-                            {% endif %}
-                        </p>
+                    {% if page.introduction %}
+                    <p class="bread-detail__introduction">
+                        {{ page.introduction }}
+                    </p>
+                    {% endif %}
```

Comment:
Format on save doesn't auto indent the templating statements, I've left this as is for now